### PR TITLE
Use tensorflow.keras in order to support eager mode and 2.0 behavior.

### PR DESCRIPTION
**Pull request by adarob**
This will make CREPE compatible with TFDS (among other newer libraries) and unblock https://github.com/tensorflow/datasets/pull/1085.

### CODE REVIEW

**Comment by adarob** [comment]
It looks like the only failure is py 3.5. Do you need to support py 3.5 (since 3.6 is passing)? If so, I can pin it to tf 1.15.

**Comment by jongwook** [comment]
Thanks for the PR and your interest in CREPE! I've previously tried replacing with `tf.keras` but had a weird accuracy drop compared to `keras`. Hopefully it doesn't matter now.

I'm happy with dropping py3.5 and even explicitly depending on TF2.0 since it has more lenient GPU package dependency.

**Comment by adarob** [comment]
Excellent! If you could make a new pip package with this change, that would be wonderful! We are finding CREPE quite useful for our projects in github.com/tensorflow/magenta

**Comment by adarob** [comment]
@jongwook do you have an ETA on when you can pull this in and make a new pip package? We may need to find another solution so we can get tensorflow/datasets#1085 in ASAP.

**Comment by jongwook** [comment]
@adarob is EOD today okay? I had wanted to verify #37 together before the next release, but I can prioritize on this given the circumstance.

**Comment by adarob** [comment]
EOD sounds good, thanks!

**Comment by jongwook** [comment]
`pip install crepe==0.0.9` should work!

**Comment by adarob** [comment]
🙏 thanks!
